package.json

```diff
@@ -53,6 +62,7 @@
     "checkLicenses": "grunt licenses",
     "build": "grunt build",
     "release": "grunt release",
+    "prettier": "prettier \"./**/*.js\" --write",
```

Comment:
I'd prefer that this was `node scripts/prettier` rather than an npm script, See https://github.com/elastic/kibana/issues/11095